OIPA/api/country/views.py

```diff
@@ -10,17 +10,67 @@
 
 
 class CountryList(ListAPIView):
+    """
+    Returns a list of IATI Countries stored in OIPA.
+
+    ## Result details
+
+    Each result item contains short information about country including URI
+    to country details.
+
```

Comment:
request parameters: 
`fields` request parameter is available here.
aggregation on activities per country is possible. `/api/countries?fields[aggregations]=total_budget`